src/org/thoughtcrime/securesms/database/RecipientDatabase.java

```diff
@@ -314,6 +325,14 @@ public void setProfileSharing(@NonNull Recipient recipient, @SuppressWarnings("S
     return results;
   }
 
+  private int processNicknameSqlRequest(ContentValues values, Address address) {
+    SQLiteDatabase db = databaseHelper.getWritableDatabase();
+    return db.update(this.TABLE_NAME, values, ADDRESS + " = ? AND " +
```

Comment:
I feel like this method call can be formatted a bit nicer.

Reply:
can you propose something ??? this method will also be used by removeNickname, and the logic that it implements is that if the new nickname is the same as the old nickname then return false. otherwise return true

Reply:
```